packages/text-field/test/text-field.test.js

```diff
@@ -166,150 +166,6 @@ describe('text-field', () => {
         expect(input.selectionEnd - input.selectionStart).to.equal(3);
       });
     });
-
-    describe('initial validation', () => {
-      let validateSpy;
-
-      beforeEach(() => {
-        textField = document.createElement('vaadin-text-field');
-        validateSpy = sinon.spy(textField, 'validate');
-      });
-
-      afterEach(() => {
-        textField.remove();
-      });
-
-      it('should not validate by default', async () => {
-        document.body.appendChild(textField);
-        await nextRender();
-        expect(validateSpy.called).to.be.false;
-      });
-
-      it('should not validate when the field has an initial value', async () => {
-        textField.value = 'Initial Value';
-        document.body.appendChild(textField);
-        await nextRender();
-        expect(validateSpy.called).to.be.false;
-      });
-
-      it('should not validate when the field has an initial value and invalid', async () => {
-        textField.value = 'Initial Value';
-        textField.invalid = true;
-        document.body.appendChild(textField);
-        await nextRender();
-        expect(validateSpy.called).to.be.false;
-      });
-    });
-
-    describe('validation', () => {
-      it('should fire a validated event on validation success', () => {
-        const validatedSpy = sinon.spy();
-        textField.addEventListener('validated', validatedSpy);
-        textField.validate();
-
-        expect(validatedSpy.calledOnce).to.be.true;
-        const event = validatedSpy.firstCall.args[0];
-        expect(event.detail.valid).to.be.true;
-      });
-
-      it('should fire a validated event on validation failure', () => {
-        const validatedSpy = sinon.spy();
-        textField.addEventListener('validated', validatedSpy);
-        textField.required = true;
-        textField.validate();
-
-        expect(validatedSpy.calledOnce).to.be.true;
-        const event = validatedSpy.firstCall.args[0];
-        expect(event.detail.valid).to.be.false;
-      });
-    });
-
-    describe('validation constraints', () => {
-      it('should not validate the field when minlength is set', () => {
-        textField.minlength = 2;
-        expect(textField.invalid).to.be.false;
-      });
-
-      it('should not validate the field when maxlength is set', () => {
-        textField.maxlength = 6;
-        expect(textField.invalid).to.be.false;
-      });
-
-      it('should validate the field when invalid after minlength is changed', () => {
-        textField.invalid = true;
-        const spy = sinon.spy(textField, 'validate');
-        textField.minlength = 2;
-        expect(spy.calledOnce).to.be.true;
-      });
-
-      it('should validate the field when invalid after maxlength is changed', () => {
```

Comment:
Was this test intentionally removed? Are there other removals?

Reply:
Ah, good catch. It was not intentional. Reverted. This test will be replaced later but not in this PR.